### PR TITLE
[Security] Mention the feature to use a custom AccessDecisionManager

### LINKED ISSUE #8149: [Security] Custom access decision manager services
see symfony/symfony#19034

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #8149.

### CODE REVIEW

File: security/voters.rst [hunk 1]
```diff
@@ -323,3 +323,45 @@ security configuration:
                 'allow_if_all_abstain' => false,
             ],
         ]);
+
+If none of the built-in strategies fits your use case, define the ``service``
```
Comment (wouterj) [inline security/voters.rst:327]: What about starting a new section?

```rst
Custom Access Decision Strategy
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: security/voters.rst [hunk 2]
```diff
@@ -323,3 +323,45 @@ security configuration:
...
+option to use a custom service as the Access Decision Manager:
```
Comment (wouterj) [inline security/voters.rst:328]: I think we should mention somewhere which interface has to be implemented by this service? (`Symfony\Component\Security\Core\Authorization\AccessDecisionManagerInterface`)

**Review by wouterj — APPROVED** [review]
Thanks for cleaning up some old issues Javier!

**Comment by javiereguiluz** [comment]
Thanks for the review! I made all the suggested changes because they are great!

**Pull request merged**